oauth2-server/src/main/java/com/clouway/oauth2/RefreshTokenActivity.java

```diff
@@ -1,21 +1,28 @@
 package com.clouway.oauth2;
 
-import com.clouway.oauth2.client.Client;
 import com.clouway.friendlyserve.Request;
 import com.clouway.friendlyserve.Response;
+import com.clouway.oauth2.client.Client;
 import com.clouway.oauth2.token.BearerToken;
+import com.clouway.oauth2.token.IdTokenFactory;
 import com.clouway.oauth2.token.TokenResponse;
 import com.clouway.oauth2.token.Tokens;
+import com.clouway.oauth2.user.IdentityFinder;
+import com.google.common.base.Optional;
 
 /**
  * @author Miroslav Genov (miroslav.genov@clouway.com)
  */
 class RefreshTokenActivity implements ClientActivity {
 
   private final Tokens tokens;
+  private final IdTokenFactory idIdTokenFactory;
+  private final IdentityFinder identityFinder;
 
-  RefreshTokenActivity(Tokens tokens) {
+  RefreshTokenActivity(Tokens tokens, IdTokenFactory idIdTokenFactory, IdentityFinder identityFinder) {
```

Comment:
Still idIdTokenFactory.

Reply:
Changed. I found a few more places that was named idId and changed them as well